src/main/java/stream/flarebot/flarebot/util/pagination/PagedTableBuilder.java

```diff
@@ -0,0 +1,135 @@
+package stream.flarebot.flarebot.util.pagination;
+
+import org.apache.commons.lang3.ArrayUtils;
+import stream.flarebot.flarebot.util.MessageUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Used for Building Paged Tables.
+ */
+public class PagedTableBuilder {
+    private List<String> header;
+    private List<List<String>> body;
+    private int rows;
+    private boolean pageCounts;
+
+    public PagedTableBuilder() {
+        body = new ArrayList<>();
+        rows = 15;
+        pageCounts = true;
+        this.header = new ArrayList<>();
+    }
+
+    /**
+     * Sets the column values
+     *
+     * @param columns A list of the column values to use for this table
+     * @return this
+     */
+    public PagedTableBuilder setColumns(List<String> columns) {
```

Comment:
Probably better to just make this an array, I don't see a use case for List

Reply:
But arrays are harder to work with 😦

Reply:
btw with arrays do you want `String...` or `String[]`

Reply:
You don't want VarArgs for this, use `String[]`

Reply:
👍